docs/examples/Ch-ExamplesAddingModbusDevice.rst

```diff
@@ -2,459 +2,273 @@
 
-    * name: edgex-device-modbus
+Once we execute a command, device-modbus knows its value type and register type, startingAddress, and register length. So it can read or write value using the modbus protocol.
 
-  * profile:
+    .. image:: properties.png
+        :scale: 50%
+        :alt: Properties
 
-    * name: mbus-rth-lcd (same as used in modbus device profile)
+|
+|
 
-* Press the “Send” button when you are finished
-* Note the addressable id
+    .. image:: holdingregisters.png
+        :scale: 70%
+        :alt: Holding Registers
 
-**What if a Mistake is Made**
+|
+|
 
-* Get device id
-* Delete device id
-* Get device profile id
-* Delete device profile id
-* Get addressable id
-* Delete addressable id
+    .. image:: profileyaml.png
+        :scale: 70%
+        :alt: Profile YAML
 
-**Verify Device Added**
 
-Check the edgex-device-modbus logs to see if the device was added without issue
+Set Up Device Service Configuration
+-----------------------------------
 
-“sudo docker logs -f --tail 100 edgex-device-modbus”
+Use this configuration file to define devices and schedule jobs. The device-modbus generates a relative instance on startup.
 
-**Creating a Scheduled Event**
+device-modbus offers two types of protocol, Modbus TCP and Modbus RTU. An addressable can be defined as shown below:
 
-This is used to regularly get & push data to another service or for regularly viewing data.
-Gathering information for the addressable
+   .. csv-table:: Modbus Protocols
+       :header: "protocol", "Name", "Protocol", "Address", "Port", "Path"
+       :widths: 20, 20, 10, 20, 10, 10
 
-Got to http://localhost:48082/api/v1/device
+       "Modbus TCP", "Gateway address 1", "TCP", "10.211.55.6", "502", "1"
+       "Modbus RTU", "Gateway address 2", "RTU", "/tmp/slave,19200,8,1,0", "502", "2"
 
-Look for the id or the device that you want to schedule an event for
+Path defines the Modbus device's unit ID (or slave ID).
 
-::
+In the RTU protocol, address is defined in five comma-separated parts:
 
-  [
+* serial port
+* baud rate 
+* data bits
+* stop bits
+* parity (N - None is 0, O - Odd is 1, E - Even is 2, default is E).
 
-     {
+    .. image:: configuration.png
+        :scale: 50%
+        :alt: configuration.toml File
 
-         "name": "mbus-rth-lcd-device",
+Add Device Service to docker-compose File
+-----------------------------------------
 
-         "id": "5a1dd585e4b0c3936013123d",  <--- This
+Because we deploy EdgeX using docker-compose, we must add the device-modbus to the docker-compose file ( https://github.com/edgexfoundry/developer-scripts/blob/master/compose-files/docker-compose-delhi-0.7.0.yml ). If you have prepared configuration files, you can mount them using volumes and change the entrypoint for device-modbus internal use.
```

Comment:
Include the text to add to docker-compose.yaml in a block quote here for easy copy/paste

Reply:
Updated